front/app/containers/Admin/reporting/components/ReportBuilderPage/CreateReportModal.tsx

```diff
@@ -81,7 +81,7 @@ const CreateReportModal = ({ open, onClose }: Props) => {
           const path = `${route}/${report.data.id}/editor`;
           const params =
             template === 'project' && selectedProject
-              ? `?projectId=${selectedProject}`
+              ? `?templateProjectId=${selectedProject}`
```

Comment:
Used for the old report templates. Renamed this to distinguish it from the other project id that we have in phase reports (from the project that the phase belongs to)